packages/frontend-2/components/settings/user/Profile.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <section>
+    <div class="md:max-w-xl md:mx-auto pb-6 md:pb-0">
+      <SettingsSectionHeader title="Profile" text="Manage your profile information" />
+      <SettingsUserProfileDetails :user="user" />
+      <hr class="my-6 md:my-10" />
+      <SettingsUserProfileChangePassword :user="user" />
+      <hr class="my-6 md:my-10" />
+      <SettingsUserProfileDeleteAccount :user="user" />
+    </div>
+  </section>
+</template>
+
+<script setup lang="ts">
+import type { User } from '~~/lib/common/generated/gql/graphql'
```

Comment:
With this type you're asking for way too much data, when you only need like 1% of it all - https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#d40e40efc32f4e5ea67b68e848aadcd0